build/code-batch-process.go

```diff
@@ -40,7 +40,7 @@ func passThroughCmd(cmd string, args []string) error {
 	}
 	c := exec.Cmd{
 		Path:   foundCmd,
-		Args:   args,
+		Args:   append([]string{cmd}, args...),
```

Comment:
I think this was previously at least partially broken the first arg should be the command again.

> Args holds command line arguments, including the command as Args[0]